From: Emmanuel Arias <eamanu@yaerobi.com>
Date: Wed, 28 Oct 2020 21:55:59 -0300
Subject: Skip test on build process

The test skipped use example folder for make the test. Examples tests is
used for docs and shouldn't have a participation on the tests build. For
this reason this test is skiped.
---
 tests/test_open_workbook.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/test_open_workbook.py b/tests/test_open_workbook.py
index e8ef978..398192c 100644
--- a/tests/test_open_workbook.py
+++ b/tests/test_open_workbook.py
@@ -1,6 +1,7 @@
 import os
 import shutil
 import tempfile
+import unittest
 from unittest import TestCase
 
 from xlrd import open_workbook
@@ -10,7 +11,7 @@ from .base import from_this_dir
 
 class TestOpen(TestCase):
     # test different uses of open_workbook
-
+    @unittest.skip("Skip test on Debian")
     def test_names_demo(self):
         # For now, we just check this doesn't raise an error.
         open_workbook(
